js/actions/bookmarkActions.js

```diff
@@ -4,15 +4,18 @@
 
 'use strict'
 
+const Immutable = require('immutable')
 const siteUtil = require('../state/siteUtil')
 const appActions = require('./appActions')
 const windowActions = require('./windowActions')
 const eventUtil = require('../lib/eventUtil')
+const appStoreRenderer = require('../stores/appStoreRenderer')
 const {SWITCH_TO_NEW_TABS} = require('../constants/settings')
 const getSetting = require('../settings').getSetting
 
 const bookmarkActions = {
-  openBookmarksInFolder: function (allBookmarkItems, folderDetail) {
+  openBookmarksInFolder: function (folderDetail) {
```

Comment:
I am not sure that that's enough. So if you check out  `createTabRequested` you will see that we have two places where we call `APP_CREATE_TAB_REQUESTED`.

First: https://github.com/brave/browser-laptop/blob/master/app/browser/reducers/tabsReducer.js?utf8=%E2%9C%93#L138

Second: https://github.com/brave/browser-laptop/blob/master/app/browser/reducers/sitesReducer.js?utf8=%E2%9C%93#L158

Now if instead of calling this app actions I just do `tabs.create` some logic will be lost, which would be called otherwise. This is the main reason why I keep using actions, because they are used in different places and I could regress some logic if I just pick one. What I don't want to do is duplicating a code.